src/index.ts

```diff
@@ -28,6 +29,7 @@ const qualities: { [K in Quality]: null } = {
   [Quality.Stable]: null,
   [Quality.Beta]: null,
   [Quality.Dev]: null,
+  [Quality.Nightly]: null,
```

Comment:
I would use `Canary` instead of making a new Nightly here. Canary is Chrome/Edge's version of "nightly" so should be functionally the same.